client/state/jetpack-sync/selectors.js

```diff
@@ -99,23 +99,23 @@ function getSyncProgressPercentage( state, siteId ) {
 	const countQueued = reduce(
```

Comment:
How about just using lodash's `sum`?

```suggestion
	const countQueued = sum( queued );
```